src/librbd/cache/PassthroughImageCache.cc

```diff
@@ -126,7 +126,7 @@ void PassthroughImageCache<I>::flush(Context *on_finish) {
 
   // internal flush -- nothing to writeback but make sure
   // in-flight IO is flushed
-  aio_flush(on_finish);
+  aio_flush(librbd::io::FLUSH_SOURCE_USER, on_finish);
```

Comment:
Nit: as the comment above states, this was for internal flushes.

Reply:
+1, The type should be FLUSH_SOURCE_INTERNAL.

Reply:
@dillaman Done.